stdlib/map.gr

```diff
@@ -274,8 +277,20 @@ export let fromList = (list) => {
   map
 }
 
+let setInArray = (array) => (i, key, value) => {
+  array[i] = (key, value)
+  i + 1
+}
```

Comment:
I'm not exactly sure why this doesn't need `@disableGC` anymore??

Reply:
We do, my mistake. It needs to go on the inner function though, and we'll need to incRef both `key` and `value` here.

Reply:
Should be good-to-go!